plugins/libplugin.c

```diff
@@ -1058,7 +1059,14 @@ static void handle_rpc_reply(struct plugin *plugin, const jsmntok_t *toks)
 	cmd_freed = false;
 	tal_add_destructor2(out->cmd, destroy_cmd_mark_freed, &cmd_freed);
 
+	trace_span_resume(out);
 	contenttok = json_get_member(buf, toks, "error");
+
+	/* Annotate the JSON-RPC span whether it succeeded or failed,
+	 * and then emit it. */
+	trace_span_tag(out, "error", contenttok == NULL ? "true" : "false");
```

Comment:
This needs to be the other way around

```suggestion
	trace_span_tag(out, "error", contenttok == NULL ? "false" : "true");
```

Reply:
That's two negatives.  I will change it  to `contenttok ? "true" : "false"`